pkg/oci/utils.go

```diff
@@ -286,6 +287,35 @@ func (spec *CompatOCISpec) PodID() (string, error) {
 	return "", fmt.Errorf("Could not find pod ID")
 }
 
+func ociResourceUpdateRuntimeConfig(ocispec CompatOCISpec, config *RuntimeConfig) error {
+	if ocispec.Linux == nil {
+		return nil
+	}
+
+	if ocispec.Linux.Resources == nil {
+		return nil
+	}
+
+	if ocispec.Linux.Resources.Memory == nil {
+		return nil
+	}
+
+	if ocispec.Linux.Resources.Memory.Limit == nil {
+		return nil
+	}
+
+	memBytes := *ocispec.Linux.Resources.Memory.Limit
+
+	// round up memory bytes to MB
+	mem := int(math.Ceil(float64(memBytes) / 1024 / 1024))
```

Comment:
@grahamwhaley yes I like this one, you add 1MiB to make sure we round this up. My main concern was to avoid playing with `float64` numbers whilst we don't __really__ need that much precision.